### PR TITLE
It's now possible to define -genconfigprofilename in the generate.js …

### PR DESCRIPTION
**Pull request by pgilmorepf**
…args to specify which profile from genConfig.json to read

### CODE REVIEW

File: SDKGenerator.njsproj [hunk 1]
```diff
@@ -18,7 +18,7 @@
     <ProjectGuid>{2ba807ef-310c-43b5-8d42-1abcfff39be9}</ProjectGuid>
     <ProjectHome>.</ProjectHome>
     <ProjectView>ShowAllFiles</ProjectView>
-    <StartupFile>generate.js</StartupFile>
+    <StartupFile>generate.ts</StartupFile>
```
Comment (toddbellMSFT) [inline SDKGenerator.njsproj:21]: OH, that's different

File: generate.js [hunk 2]
```diff
@@ -1,3 +1,6 @@
+/// <reference path="node.d.ts"/>
```
Comment (toddbellMSFT) [inline generate.js:1]: Should these comments be left in?

File: generate.ts [hunk 3]
```diff
@@ -1,3 +1,7 @@
+/// <reference path="node.d.ts"/>
```
Comment (toddbellMSFT) [inline generate.ts:1]: same comment from previous file, are these comments needed

File: SDKGenerator.njsproj (hunk 1 above)
Comment (pgilmorepf) [reply SDKGenerator.njsproj:21]: The sln seems to not create the js file properly.
I was experimenting.
VS should understand what .ts files are though, and it built correctly after I manually regenerated the .js files using tsc.

**Pull request merged**